tests/validation/tests/v3_api/test_upgrade.py

```diff
@@ -70,10 +70,13 @@
 app_validate_name = validate_prefix + "-app"
 # the pre_upgrade_externalId is for launching an app
 pre_upgrade_externalId = \
-    "catalog://?catalog=library&template=mysql&version=0.3.7"
+    "catalog://?catalog=test-catalog&template=mysql&version=1.3.1"
 # the post_upgrade_externalId is for upgrading the existing app
 post_upgrade_externalId = \
-    "catalog://?catalog=library&template=mysql&version=0.3.8"
+    "catalog://?catalog=test-catalog&template=mysql&version=1.3.2"
+
+catalogUrl = "https://github.com/sowmyav27/rancher-charts.git"
```

Comment:
Ultimately, I would like to see us have a repo for test-charts hosted in Rancher github, rather than using from a personal fork, but we can do this later if necessary.

Reply:
done